main.go

```diff
@@ -220,15 +222,24 @@ func run(state overseer.State) {
 		if len(*githubScanOrgs) == 0 && len(*githubScanRepos) == 0 {
 			logrus.Fatal("You must specify at least one organization or repository.")
 		}
+		var repos []string
+		// If an org is provided, the repos would be the IncludeRepos.
+		// If an org is not provided, the repos would be the repos passed in via the --repo flag.
+		if len(*githubScanOrgs) == 0 {
```

Comment:
If a user specifies an `org` and a `repo` they should both be included.

Reply:
oh, interesting. I didn't even realize we supported both during the same command. good to know. Will update this.